src/app/tests.py

```diff
@@ -1543,3 +1544,27 @@ def test_no_license_id_on_license_requests_edit_xml(self):
         self.assertEqual(resp.status_code,200)
         self.assertIn("app/license_namespace_requests.html",(i.name for i in resp.templates))
         self.assertEqual(resp.resolver_match.func.__name__,"licenseNamespaceRequests")
+
+class TestCronJob(TestCase):
+    def test_delete_old_files(self):
```

Comment:
We can directly run test on cleanup.py script

Reply:
Ok, I will change the unit test as well, actually currently it was being run through a django-admin command, I will change that

Reply:
I have changed the tests now, they now directly use the cleanup.py script instead of the django commands
![image](https://user-images.githubusercontent.com/65075935/227784660-b8fb95d3-d62f-4948-a015-818d8fbf1299.png)